### PR TITLE
Added support policy

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Content based on https://help.stackrox.com/docs/system-info/supported-platforms/

Preview:
- https://openshift-docs-git-support-policy-gnelson.vercel.app/openshift-acs/master/support/getting-support-stackrox.html
- https://openshift-docs-git-support-policy-gnelson.vercel.app/openshift-acs/master/support/getting-support.html (I've used the OpenShift Support assembly and modules for this one)

We want to include old support related information along with the new one, therefore, I am using the old product name for one of the pages.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@pneedle-rh @bobfuru Can any of you review this one?

Thank you 👍🏼

File: modules/stackrox-kubernetes-security-platform-support.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+StackRox supports the StackRox Kubernetes Security Platform versions for up to 6 months after its release, which corresponds to the previous 9 released versions. StackRox will make reasonable efforts and assist you in supporting some older versions. However, the support team may request you to upgrade to a newer released version of the StackRox Kubernetes Security Platform for full support.
+
+[NOTE]
+====
+For the StackRox Kubernetes Security Platform, StackRox supports:
+
+* The latest released version (which we’ll refer to as N).
+* 9 earlier versions.
+
+The support window of the StackRox Kubernetes Security Platform versions is known as N-9: (N (Latest release) - 9 (earlier versions)).
```
Comment (bobfuru) [inline modules/stackrox-kubernetes-security-platform-support.adoc:18]: It's okay to use the numeral "9" in these references. The outer parentheses should be square brackets, so:
[N (Latest release) - 9 (earlier versions)]. However, I think we can reduce this confusion further by the addition of "where". For example:

> The support window of the StackRox Kubernetes Security Platform versions is known as N-9 
> where: 
> N (latest release) - 9 (earlier versions)

ISG gives several examples of using the word _where_ with a lowercase _w_ to introduce an explanation.

Or you could just use it in a running sentence: 

> The support window of the StackRox Kubernetes Security Platform versions is known as N-9, where: N (latest release) - 9 (earlier versions).

(Also, note the lowercase _l_ in _latest_.)

File: modules/stackrox-kubernetes-security-platform-support.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Along with our N-9 support window, StackRox may support version N-10 to N-13, depending on a case-by-case basis. StackRox will not support any earlier versions than N-13.
```
Comment (bobfuru) [inline modules/stackrox-kubernetes-security-platform-support.adoc:20]: s/may/might/ ? (ISG)

File: modules/support-knowledgebase-about.adoc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support.adoc
+
+[id="support-knowledgebase-about_{context}"]
+= About the Red Hat Knowledgebase
+
+The link:https://access.redhat.com/knowledgebase[Red Hat Knowledgebase] provides rich content aimed at helping you make the most of Red Hat's products and technologies. The Red Hat Knowledgebase consists of articles, product documentation, and videos outlining best practices on installing, configuring, and using Red Hat products. In addition, you can search for solutions to known issues, each providing concise root cause descriptions and remedial steps.
```
Comment (bobfuru) [inline modules/support-knowledgebase-about.adoc:8]: s/Red Hat's/Red Hat

File: modules/support-on-various-platforms.adoc [hunk 4]
```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support-stackrox.adoc
+:_module-type: CONCEPT
+[id="support-on-various-platforms_{context}"]
+= Support on various platforms
+
+[role="_abstract"]
+StackRox support team supports a platform version based on the upstream (or vendor) product’s support lifecycle for that version. When a platform version support reaches its end-of-life (EOL) or is not actively maintained, StackRox no longer support it.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:9]: "The StackRox support team supports"

Comment (bobfuru) [reply modules/support-on-various-platforms.adoc:9]: "When a platform version support" doesn't make sense to me. Maybe, "When support of a platform reaches..."?

s/end-of-life/end of life (The hyphens are only needed if adjectivally describing a noun, which in this case it's not 😁 )
s/no longer support it/no longer supports it

File: modules/support-on-various-platforms.adoc [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+[WARNING]
+====
+StackRox do not support:
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:13]: s/do not/does not

File: modules/support-on-various-platforms.adoc [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+* Installing the StackRox Kubernetes Security Platform on Minikube and other similar single-node clusters.
+* Amazon Elastic File System (Amazon EFS), use the Amazon Elastic Block Store (Amazon EBS) with the default GP2 volume type instead.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:16]:
```suggestion
* Amazon Elastic File System (EFS). Use Amazon Elastic Block Store (EBS) with the default GP2 volume type instead.
```

File: modules/support-on-various-platforms.adoc [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+* Older CPUs that don’t have the Streaming SIMD Extensions (SSE) 4.2 instruction set, for example Intel processors older than Sandy Bridge and AMD processors older than Bulldozer. Both of these processors were released in 2011.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:17]: spell out "do not"

File: modules/support-on-various-platforms.adoc [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+====
+
+[id="operating-systems_{context}"]
+== Operating systems
+
+|===
+| Operating system | Version
+
+| http://releases.ubuntu.com/[Ubuntu]
+| 16.04 LTS, 18.04 LTS, and 20.04 LTS with standard or cloud-provider-specific kernel versions.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:27]: no period needed at the end of the sentence

File: modules/support-on-various-platforms.adoc [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+== Container orchestrators and platforms
+
+|===
+| Container orchestrators and platforms | Version
+
+| https://kubernetes.io/docs/setup/release/notes/[Kubernetes]^1^
+| 1.15 and newer
+
+| https://access.redhat.com/support/policy/updates/openshift[{ocp}]
+| 3.10, 3.11, 4.1 and newer
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:82]: s/4.1 and newer/4.1, and newer

File: modules/support-on-various-platforms.adoc [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+
+| https://docs.d2iq.com/mesosphere/dcos/services/kubernetes/[DC/OS Kubernetes]^2^
+| 2.0.0 and newer
+|===
+
+[NOTE]
+====
+
+. StackRox support new versions of Kubernetes within 3 months of their open-source general-availability release.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:91]: - s/StackRox support/StackRox supports
- s/3 months/three months
- s/their/its

File: modules/support-on-various-platforms.adoc [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+You may need to upgrade the {product-title} to get support for the latest Kubernetes versions.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:92]: s/You may need to upgrade the {product-title}/You might need to upgrade {product-title}

File: modules/support-on-various-platforms.adoc [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+See the <<kubernetes-support,Kubernetes support>> section for more details.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:93]: This link doesn't resolve itself. Not sure what is meant by "Kubernetes support"?

File: modules/support-on-various-platforms.adoc [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+. DC/OS support is available starting from {product-title} version 2.5.31.0.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:94]: s/version 2.5.31.0/2.5.31.0/ (drop "version")

File: modules/support-on-various-platforms.adoc [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+====
+
+[id="kubernetes-support_{context}"]
+=== Kubernetes support
+
+The following table lists the minimum version of {product-title} StackRox support based on the Kubernetes version.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:100]: Double-checking that you want this to say "Red Hat Advanced Cluster Security for Kubernetes StackRox support"? Having both product names together seems confusing.

File: modules/support-on-various-platforms.adoc [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+StackRox support recent Kubernetes and {ocp} versions, and test on managed Kubernetes service from all major cloud providers, including:
+
+* https://aws.amazon.com/eks/[Amazon Elastic Kubernetes Service (EKS)]
+* https://azure.microsoft.com/en-us/services/kubernetes-service/[Azure Kubernetes Service (AKS)]
+* https://cloud.google.com/kubernetes-engine/[Google Kubernetes Engine (GKE)]
+
+[NOTE]
+====
+
+Collector will not install on GKE clusters if you enable https://cloud.google.com/kubernetes-engine/docs/how-to/shielded-gke-nodes#secure_boot[secure boot] because third-party unsigned (not signed by Google's CA) kernel modules cannot be loaded when secure boot is enabled.
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:127]: I would avoid the parentheses and say something like, "...third-party unsigned kernel module, which are not signed by Google's CA, cannot be loaded..."

File: modules/support-on-various-platforms.adoc [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+====
+
+Along with other types of clusters, StackRox also support clusters created by using the https://github.com/kubernetes/kops[kops - Kubernetes Operations] tool with the default configurations on Amazon Web Services (AWS).
```
Comment (bobfuru) [inline modules/support-on-various-platforms.adoc:130]: s/support/supports

File: modules/support-submitting-a-case.adoc [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+// * support/getting-support.adoc
+
+[id="support-submitting-a-case_{context}"]
+= Submitting a support case
+
+.Prerequisites
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You have a Red Hat Customer Portal account.
+* You have a Red Hat standard or premium Subscription.
```
Comment (bobfuru) [inline modules/support-submitting-a-case.adoc:12]: "subscription" lowercase

File: modules/support-submitting-a-case.adoc [hunk 18]
```diff
@@ -0,0 +1,44 @@
...
+[source,terminal]
+----
+$ oc get clusterversion -o jsonpath='{.items[].spec.clusterID}{"\n"}'
+----
+. Complete the following questions where prompted and then click *Continue*:
++
+* Where are you experiencing the behavior? What environment?
+* When does the behavior occur? Frequency? Repeatedly? At certain times?
+* What information can you provide around time-frames and the business impact?
+. Upload relevant diagnostic data files and click *Continue*. It is recommended to include data gathered using the `oc adm must-gather` command as a starting point, plus any issue specific data that is not collected by that command.
```
Comment (bobfuru) [inline modules/support-submitting-a-case.adoc:42]: s/issue specific data/issue-specific data

File: support/getting-support-stackrox.adoc [hunk 19]
```diff
@@ -0,0 +1,21 @@
...
+:context: getting-support-stackrox
+
+toc::[]
+
+[role="_abstract"]
+This topic provides information about the technical support for the StackRox Kubernetes Security Platform and details about other supported software and infrastructure.
+
+[IMPORTANT]
+====
+* The support releated information listed on this page is only applicable if you are not a Red Hat customer and you purchased the Stackrox Kuberentes Security Platform before the acquisition. Red Hat will honor the StackRox support policy for its duration.
```
Comment (bobfuru) [inline support/getting-support-stackrox.adoc:13]: s/The support releated information/The support-related information

File: support/getting-support.adoc [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+include::modules/common-attributes.adoc[]
+:context: getting-support
+
+toc::[]
+
+This topic provides information about the technical support for {product-title}.
+
+[IMPORTANT]
+====
+* The support releated information listed on this page is only applicable if you are a Red Hat customer.
```
Comment (bobfuru) [inline support/getting-support.adoc:12]: s/The support releated information/The support-related information

**Review by bobfuru — COMMENTED** [review]
In the interest of time, I didn't look up references to some of ISG references here but if you'd like additional clarification, let me know. Much of the comments are related to how we should refer to company and product in the third-person. Example, "StackRox supports" instead of "StackRox" support..."

File: support/getting-support-stackrox.adoc [hunk 21]
```diff
@@ -0,0 +1,21 @@
...
+:context: getting-support-stackrox
+
+toc::[]
+
+[role="_abstract"]
+This topic provides information about the technical support for the StackRox Kubernetes Security Platform and details about other supported software and infrastructure.
+
+[IMPORTANT]
+====
+* The support-releated information listed on this page is only applicable if you are not a Red Hat customer and you purchased the Stackrox Kuberentes Security Platform before the acquisition. Red Hat will honor the StackRox support policy for its duration.
```
Comment (bobfuru) [inline support/getting-support-stackrox.adoc:13]: I'll fix this typo in a follow-up PR.

File: support/getting-support.adoc [hunk 22]
```diff
@@ -0,0 +1,30 @@
...
+include::modules/common-attributes.adoc[]
+:context: getting-support
+
+toc::[]
+
+This topic provides information about the technical support for {product-title}.
+
+[IMPORTANT]
+====
+* The support-releated information listed on this page is only applicable if you are a Red Hat customer.
```
Comment (bobfuru) [inline support/getting-support.adoc:12]: I'll fix this typo in a follow-up PR.

**Review by bobfuru — COMMENTED** [review]
Overall, lgtm. A few minor typos that I'll fix and merge in a follow-up PR.

File: support/getting-support-stackrox.adoc (hunk 21 above)
Comment (bobfuru) [reply support/getting-support-stackrox.adoc:13]: https://github.com/openshift/openshift-docs/pull/34749

File: support/getting-support.adoc (hunk 22 above)
Comment (bobfuru) [reply support/getting-support.adoc:12]: https://github.com/openshift/openshift-docs/pull/34749

File: modules/stackrox-kubernetes-security-platform-support.adoc [hunk 23]
```diff
@@ -0,0 +1,21 @@
...
+= StackRox Kubernetes Security Platform support
+
+[role="_abstract"]
+StackRox supports the StackRox Kubernetes Security Platform versions for up to six months after its release, which corresponds to the previous nine released versions. StackRox will make reasonable efforts and assist you in supporting some older versions. However, the support team may request you to upgrade to a newer released version of the StackRox Kubernetes Security Platform for full support.
+
+[NOTE]
+====
+For the StackRox Kubernetes Security Platform, StackRox supports:
+
+* The latest released version (reffered as N).
```
[the remark this answers is not in the document]
Comment (bobfuru) [reply modules/stackrox-kubernetes-security-platform-support.adoc:15]: https://github.com/openshift/openshift-docs/pull/34749

**Pull request merged**